user/tests/app/interrupts/interrupts.cpp

```diff
@@ -0,0 +1,271 @@
+#include "Particle.h"
```

Comment:
Let's turn this into a test that uses our testing framework? We can then run this on HIL if required with a fixture. Moreover there is no user interaction here, so another argument for not having this as a regular application.

Reply:
There is user interaction because you have to move the wire from A5 to D0, then A5 to D1... etc.. resetting or re-running the test each time.  This test has served its purpose of testing all platforms for this issue as it stands.  It's easy to compile and run.  I don't think we need to reconfigure it for a "fixtured" test framework until we have that framework in place.  Right now all "fixtured" tests are run manually.

Reply:
Ok, fair enough. Even though we have examples with user interaction (wiring/sleep20) this is more akin to `sleep_multiple_pins`, so let's leave as-is :+1:

(Ugh, wish we had something like a CPLD/FPGA that we could control from test runner to dynamically 'connect' pins at least for digital IO)

Reply:
Yeah, we'll get there and can then automate all of these manual tests 👍🏼 💯